docs/index.md

```diff
@@ -72,25 +70,59 @@ print(Settings().model_dump())
 """
 ```
 
+1. The environment variable name is overridden using `validation_alias`. In this case, the environment variable
+   `my_auth_key` will be read instead of `auth_key`.
+
+    Check [`Field`](TODO: Add link when available) for more information.
+
+2. The `AliasChoices` class allows to have multiple environment variable names for a single field.
+   The first environment variable that is found will be used.
+
+    Check [`AliasChoices`](TODO: Add link when available) for more information.
+
+3. The `ImportString` class allows to import an object from a string.
+   In this case, the environment variable `special_function` will be read and the function `math.cos` will be imported.
+
+4. The `env_prefix` config setting allows to set a prefix for all environment variables.
+
+    Check [Environment variable names](#environment-variable-names) for more information.
+
 ## Environment variable names
 
-The following rules are used to determine which environment variable(s) are read for a given field:
+By default, the environment variable name is the same as the field name.
+
+You can change the prefix for all environment variables by setting the `env_prefix` config setting:
+
+```py
+from pydantic import ConfigDict
+
+from pydantic_settings import BaseSettings
+
+
+class Settings(BaseSettings):
+    model_config = ConfigDict(env_prefix='my_prefix_')
+
+    auth_key: str = 'xxx'  # will be read from `my_prefix_auth_key`
+```
+
+!!! note
+    The default `env_prefix` is `''` (empty string).
+
+If you want to change the environment variable name for a single field, you can use an alias.
+
+There are two ways to do this:
 
-* By default, the environment variable name is built by concatenating the prefix and field name.
-  * For example, to override `special_function` above, you could use:
+* Use `Field(validation_alias=...)` (see `auth_key` above)
+* Use `Field(alias=...)` (see `api_key` above)
```

Comment:
```suggestion
* Using `Field(alias=...)` (see `api_key` above)
* Using `Field(validation_alias=...)` (see `auth_key` above)
```
I changed the order since it seems to me that using `alias` would be more common.